### PR TITLE
add eastbio workshop

### PR DESCRIPTION
**Pull request by jburel**
Add entry to the upcoming eastbio workshop
cc @pwalczysko

### CODE REVIEW

File: events/omero-workshop-eastbio-dundee-november-2021.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organiser: <a href="http://www.eastscotbiodtp.ac.uk/" target="_blank">EastBio</a></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: Dundee</h2>
```
Comment (pwalczysko) [inline events/omero-workshop-eastbio-dundee-november-2021.html:20]:
```suggestion
                <h2 class="event-day">Location: online</h2>
```

suggesting "online" as this was the formulation on https://www.openmicroscopy.org/events/omero-users-workshop-eastbio-edinburgh-online-november-2020.html. More precisely, it is held from Edinburgh Uni remotely, but "online" should do imho

**Comment by pwalczysko** [comment]
there is no entry on the events page https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

**Comment by pwalczysko** [comment]
2 issues:
1. one minor reformulation
2. missing entry on Events (major)

**Comment by sbesson** [comment]
Unrelated to this PR, the requirement to update manually the `events/index.html` page is quite tedious. Maybe a RFE would be to rethink the current strategy and 1- either treat `events` as posts with a specific category (like `blog`) or 2- define a new collection like `minutes`

**Comment by jburel** [comment]
> Unrelated to this PR, the requirement to update manually the `events/index.html` page is quite tedious. Maybe a RFE would be to rethink the current strategy and 1- either treat `events` as posts with a specific category (like `blog`) or 2- define a new collection like `minutes`

Agree, I did not have the time to look at it so I went for the short time solution.

**Comment by sbesson** [comment]
> Agree, I did not have the time to look at it so I went for the short time solution.

Should I turn this into an issue?

**Comment by jburel** [comment]
> > Agree, I did not have the time to look at it so I went for the short time solution.
> 
> Should I turn this into an issue?

yes please.

**Comment by jburel** [comment]
The failure is not related to the change in this PR. but the document https://www.upf.edu/documents/4181972/7974943/Rocket%40OME2017/3d9cebf8-2c95-5952-c458-cc4f9908756d does not seem to be there anymore

**Comment by jburel** [comment]
b0d09878913b5007851a64892e314783afccd87c is not really part of this workflow, mainly to get the build passing

**Comment by pwalczysko** [comment]
LGTM, the https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ has the new item.

**Pull request merged**